dev/io.openliberty.microprofile.telemetry.internal.common/src/io/openliberty/microprofile/telemetry/internal/common/info/OpenTelemtryLifecycleManagerImpl.java

```diff
@@ -86,6 +86,10 @@ private void checkThenSetRuntimeFields(boolean telemetry2OrLater) {
         HashMap<String, String> propreties = OpenTelemetryPropertiesReader.getRuntimeInstanceTelemetryProperties();
         isRuntimeEnabled = telemetry2OrLater && !!!OpenTelemetryPropertiesReader.checkDisabled(propreties);
 
+        if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+            Tr.debug(tc, "Configured runtime mode as ", isRuntimeEnabled);
+        }
```

Comment:
```suggestion
        if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
            Tr.debug(this, tc, "Configured runtime mode as ", isRuntimeEnabled);
        }
```

Needed put the correct value in the `id=xxxxxxxx` field in the trace log. All trace calls in non-static methods should pass `this` as the first argument.